xilem_web/src/concurrent/memoized_await.rs

```diff
@@ -84,20 +169,78 @@ where
     State: 'static,
     Action: 'static,
     Data: PartialEq + 'static,
-    FOut: std::fmt::Debug + 'static,
+    FOut: fmt::Debug + 'static,
     F: Future<Output = FOut> + 'static,
     InitFuture: Fn(&Data) -> F + 'static,
     OA: OptionalAction<Action> + 'static,
     Callback: Fn(&mut State, FOut) -> OA + 'static,
 {
-    MemoizedAwait {
+    MemoizedAwait(MemoizedInner {
         init_future,
         data,
         callback,
         debounce_ms: 0,
         reset_debounce_on_update: true,
         phantom: PhantomData,
-    }
+    })
+}
+
+/// Await a stream returned by `init_stream` invoked with the argument `data`, `callback` is called with the items of the stream.
+/// `init_stream` will be invoked again, when `data` changes.
+///
+/// The update behavior can be controlled, by [`debounce_ms`](`MemoizedStream::debounce_ms`) and [`reset_debounce_on_update`](`MemoizedStream::reset_debounce_on_update`)
```

Comment:
```suggestion
/// The behavior of the `init_stream` invocation by changes to `data` can be customized by [`debounce_ms`](`MemoizedStream::debounce_ms`) and [`reset_debounce_on_update`](`MemoizedStream::reset_debounce_on_update`).
```

(maybe also something like this for `MemoizedAwait` as I think it's not entirely clear what I meant with "update behavior").